lua/lsp-file-operations.lua

```diff
@@ -1,58 +1,86 @@
 local M = {}
 
-local will_rename = require('lsp-file-operations.will-rename')
-local log = require('lsp-file-operations.log')
+local log = require("lsp-file-operations.log")
 
 local default_config = {
   debug = false,
   timeout_ms = 10000,
 }
```

Comment:
I would add an option to configure which events user wants to subscribe to. To allow them to opt out if it causes problems.
It was not needed before since we had only one event and if you don't want to subscribe to it you can just delete the plugin.

Reply:
Yeah that's a great idea! I just pushed a commit that makes it easy to toggle operations on/off. Let me know what you think!